tools/openshift/ocp-ipi-aws/ocp-ipi-aws-prep/machine-set-resources.tf

```diff
@@ -61,3 +86,15 @@ resource "local_file" "machine_set_config" {
     "null_resource.empty_file",
   ]
 }
+
+# Create machine set config file from template.
+resource "local_file" "machine_set_config_ha" {
+  count = var.enable_ha ? 1 : 0
```

Comment:
where is the count used?

Reply:
This is not to create the resource if enable_ha is false. If the resource count is zero no resource will be created. This is a way to implement if-else in terraform. I assume this count is used internally by terraform

Reply:
Okay, thanks @aswinsuryan